tests/esm/package.json

```
@@ -0,0 +1,5 @@
+{
```

Optional; probably not-needed because there’s no package name. But this could save an accidental publish:

```suggestion
{
  "private": true
```